### PR TITLE
fix subject add bug

### PR DESCRIPTION
**Pull request by jgraff2**

### CODE REVIEW

File: MultiTable/src/RealtimeChangeHandler.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ function onSampleUpdate(data, change) {
 function onSubjectAdd(data, subject) {
```
Comment (iamigo) [inline MultiTable/src/RealtimeChangeHandler.js]: should we re-version as 1.0.6 for this fix?

Comment (iamigo) [inline MultiTable/src/RealtimeChangeHandler.js]: did you already try installing this in prod?

Comment (jgraff2) [inline MultiTable/src/RealtimeChangeHandler.js]: ok. I updated lens.json.

Comment (jgraff2) [inline MultiTable/src/RealtimeChangeHandler.js]: I don't know how to install in prod, or if I have access. You showed me briefly when I started but not in depth.

File: MultiTable/lens.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "1.0.2": "Don't use experimental Object.values(...)",
     "1.0.3": "Show messageCode even when status=OK",
     "1.0.4": "ShowAll url parameter",
-    "1.0.5": "Split overflowing groups"
+    "1.0.5": "Split overflowing groups",
+    "1.0.6": "Subject add bug fix"
   }
 }
```
Comment (iamigo) [inline MultiTable/lens.json]: Thx. Need to also regenerate the zip after updating the lens.json.

Comment (jgraff2) [inline MultiTable/lens.json]: oh, right. Done.

**Pull request merged**